index.bs

```diff
@@ -1168,7 +1168,7 @@ Each EMCAScript [=Realm=] has a corresponding <dfn>handle object map</dfn>. This
 strong [=/map=] from handle ids to their corresponding objects.
 
 <pre class="cddl remote-cddl">
-OwnershipModel = "root" / "none";
+ResultingOwnership = "root" / "none";
```

Comment:
This definition appears to be unused and can be removed.

Reply:
That works too, will do.